bundles/framework/divmanazer/component/Grid.js

```diff
@@ -1049,6 +1035,26 @@ Oskari.clazz.define('Oskari.userinterface.component.Grid',
             }
         },
 
+        updateHeaderHeight: function(){
+            var me = this;
+            // autosize header
+            if(typeof me.autoHeightHeader === 'number' && me.table) {
```

Comment:
Reverse this if so instead of having 10+ rows of code inside the method just "fails fast"/returns at the beginning of the function if there's nothing to be done for the header.